src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="calculator bg-second d-flex flex-column">
+          <div className="bg-first viewer"> </div>
+          <div className="d-flex m-1">
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">AC</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">+/-</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">%</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">/</div>
+          </div>
+          <div className="d-flex m-1">
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">7</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">8</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">9</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">X</div>
+          </div>
+          <div className="d-flex m-1">
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">4</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">5</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">6</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">-</div>
+          </div>
+          <div className="d-flex m-1">
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">1</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">2</div>
+            <div className="btn-rectingle text-first d-flex justify-content-center align-items-center">3</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">+</div>
+          </div>
+          <div className="d-flex m-1">
+            <div className="btn-rectingle btn-zero text-first d-flex justify-content-center align-items-center">0</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">.</div>
+            <div className="btn-rectingle text-white d-flex justify-content-center align-items-center">=</div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
```

Comment:
- [ ] Wow you have done an incredible job in your design, the calculator looks really awesome 👏 , However, it seems that for now, the project requirements state that we should stick to the given design, see screenshot below 👇 
![image](https://user-images.githubusercontent.com/55465133/134418589-54e12364-1cbd-4044-b2ad-57c75b8edddb.png)

- This is how yours looks like for now 👇 
![image](https://user-images.githubusercontent.com/55465133/134418702-ac4cd85e-14e1-4c87-96ac-f8af9952c33f.png)

As you can see there is a big difference, I would suggest that you start working with the design provided in the project requirements, then at the end of the last project in this series, you can change the design to match what you really like 😉 , but congratulations so far, the design was awesome 👏

Reply:
@barackm  Thank you for the Review,
The Problem with the live link is that I have not deployed the project yet, and I will do it next milestones, I deleted the live link part and wrote coming soon instead.
About the Design, My Design has the same structure as the wireframe, Every component is in its own place, the only difference is the color which I don't think will affect the Project.

Reply:
Hi @ahzia, no worries if the project is not yet deployed feel free to do so, for now, I'm changing that requirement to be optional. sorry for the delay 😉